.github/dependabot.yml

```diff
@@ -4,7 +4,9 @@ updates:
     directory: "/"
     schedule:
       interval: "monthly"
-  - package-ecosystem: "gomod"  # Dependencies listed in go.mod
-    directory: "/" # Location of package manifests
```

Comment:
does `yamllint` not like these? I don't recall seeing it complain before for inline comments, and not sure these comments are worth their own line as it adds noise...

Reply:
Yamllint need at least two whitespaces between content and comment.
I misunderstood it , now I can keep it on the same line